charts/tempo-distributed/templates/compactor/deployment-compactor.yaml

```diff
@@ -68,7 +68,7 @@ spec:
           ports:
             - containerPort: 3100
               name: http-metrics
-            - containerPort: 7946
+            - containerPort: {{ include "tempo.memberlistBindPort" . }}
```

Comment:
Hey there, have you tested it locally ?

Reply:
No I haven't, how can I run it locally ?
I did test the port change from 7946 to 7947 using kustomize and it did work, but we didn't want to keep patching the charts and would like to change the port number in one single place and have propagate to all components.

Reply:
> how can I run it locally ?

If you have minikube installed you could deploy and test the chart locally.